packages/styles/scss/components/data-table/sort/_data-table-sort.scss

```
@@ -245,11 +244,15 @@
 
   // AILabel styles
   .#{$prefix}--table-sort__header--slug .#{$prefix}--table-sort__icon,
-  .#{$prefix}--table-sort__header--slug .#{$prefix}--table-sort__icon-unsorted {
+  .#{$prefix}--table-sort__header--slug .#{$prefix}--table-sort__icon-unsorted,
+  .#{$prefix}--table-sort__header--decorator .#{$prefix}--table-sort__icon,
+  .#{$prefix}--table-sort__header--decorator
+    .#{$prefix}--table-sort__icon-unsorted {
     margin-inline: auto convert.to-rem(8px);
   }
   .#{$prefix}--table-sort__header--slug .#{$prefix}--ai-label,
-  .#{$prefix}--table-sort__header--slug .#{$prefix}--slug {
+  .#{$prefix}--table-sort__header--slug .#{$prefix}--slug,
+  .#{$prefix}--table-sort__header--decorator .#{$prefix}--tooltip {
```

since we don't know what will be in the `decorator` prop there should be some inner wrapper to target something like: 
```suggestion
  .#{$prefix}--table-sort__header--decorator .#{$prefix}--table-sort__inner-header--decorator > * {
```

---

Thanks for suggesting this!